lms/templates/courseware/course_about.html

```diff
@@ -218,6 +219,7 @@ <h1>
         ${get_course_about_section(request, course, "overview")}
       </div>
   </div>
+  </%block>
```

Comment:
There's an indentation issue here.  The `</div>` on line 221 closes the `div.details` on L211, not the `div.container` on L210.  Move the `<%block name="course_about_details">` inside the `div.container`

Reply:
Thanks for noticing it. The original `</div>` already had the wrong indentation. I corrected it and also moved the `%block` inside the `div.container` as suggested.